Resources/config/default_settings.yml

```diff
@@ -1,6 +1,8 @@
 parameters:
     ez_platformui.default.yui: {}
     ez_platformui.default.yui.filter: "min"
+    ez_platformui.default.yui.combine: true
+    ez_platformui.default.yui.build_dir: /bundles/ezplatformuiassets/vendors/yui3/build/
```

Comment:
Hmmm, odd question maybe, but why do we need a siteaccess aware setting here ? PlatformUI itself isn't really siteaccess aware...